docker/init_debian.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+# Note: When modifying this file, increment the Dockerfile-version minor version number
+# This is needed for mob tool to be able to pull the right image from the farm,
+# if it isn't done, bad things will happen to local builds and CI using mob tool
+# See mob tool comments for extended discussion
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+set -e  # Die on any errors
+
+cd /tmp
+
+# Certain Installers make 'installations' easier by having a nice front-end. While this is great when you have a manual install, this becomes an issue during automated installations.
+export DEBIAN_FRONTEND=noninteractive
+
+# Install build tools and dependencies
+apt-get update -q -q
+apt-get upgrade --yes
+apt-get install --yes \
```

Comment:
is there a good reason to remove the `--force-yes` lines?

Reply:
It is both deprecated and dangerous, e.g. https://serverfault.com/questions/1073792/apt-get-throwing-warning-w-force-yes-is-deprecated-use-one-of-the-options

I removed it because it was disabled in Ubuntu 20.04, which I have on the cloud VM I'm testing with.